index.html

```diff
@@ -58,7 +58,7 @@ <h1 id="logo"><a href="#">The PiVPN Project</a></h1>
 						<header class="major">
 							<h2>PiVPN</h2>
 							<p>
-								Simplest OpenVPN setup and configuration,<br />
```

Comment:
and here maybe ... 
"The simplest way to setup and configure a VPN"

Reply:
"setup and manage"

Reply:
done